flink-batch-connectors/flink-jdbc/src/main/java/org/apache/flink/api/java/io/jdbc/split/JDBCInputSplitsGenerator.java

```diff
@@ -15,17 +15,14 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.flink.api.java.io.jdbc;
+package org.apache.flink.api.java.io.jdbc.split;
 
-import java.io.OutputStream;
+import java.io.Serializable;
 
-@SuppressWarnings("unused")
-/**
- * Utility class to disable derby logging
- */
-public class DerbyUtil {
-	public static final OutputStream DEV_NULL = new OutputStream() {
-		public void write(int b) {
-		}
-	};
+import org.apache.flink.api.java.io.QueryParamInputSplit;
+
+public interface JDBCInputSplitsGenerator extends Serializable {
```

Comment:
I think we should hide the concept of input splits from users. Can you rename the interface to `ParameterValueProvider`?